renderer.php

```diff
@@ -649,7 +652,21 @@ protected function render_attendance_take_list(attendance_take_data $takedata) {
         foreach ($extrasearchfields as $field) {
             $row->cells[] = '';
         }
-        $row->cells[] = html_writer::div(get_string('setallstatuses', 'attendance'), 'setallstatuses');
+
+        if (get_user_preferences('mod_attendance_statusdropdown') == 'unselected') {
```

Comment:
might be good to drop this into a helper function as it's used twice in the code - we should also be using lang strings for the "all" and "unselected" items shouldn't we?